lib/android-helpers.js

```diff
@@ -628,7 +628,12 @@ helpers.verifyUnlock = async function (adb) {
 };
 
 helpers.initDevice = async function (adb, opts) {
+  if (opts.skipDeviceInitialization) {
+    logger.info(`'skipDeviceInitialization' is set. Skipping device initialization.`);
+    return await adb.defaultIME();
+  }
   await adb.waitForDevice();
```

Comment:
I think it is still necessary to wait for device and to start logcat even if the other stuff is skipped

Reply:
I thought so either, but I executed few tests with skips for all and it worked. I can perform additional tests (on other devices). Or just add that if it's really required.

Reply:
I've checked waitForDevice implementation and everything it does is the execution of wait-for-device and ping commands. I would say it is safe to skip it

Reply:
Great, thanks! So I left `waitForDevice` skipped (when `skipDeviceInitialization` is set) and restored `adb.startLogcat` to be executed independently.